content/python/concepts/time-module/terms/time-strftime/time-strftime.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'time.strftime()'
```

Comment:
Either way works here, I think, because you can introduce `strftime()` (or most modules?) in two ways:

```
from time import strftime
# "strftime()"

import time
# ".strftime()"